docs/usage/import-automated.md

```diff
@@ -96,21 +96,31 @@ CSV feed ingester enables users to import CSV files exposed on URLs. Here's a st
 
 Additional configuration options:
 
-- User responsible for data creation: Define the user responsible for creating data received from this RSS feed. Best practice is to dedicate one user per source for organizational clarity. Please [see the section "Best practices" below](import-automated.md#best-practices-section) for more information.
+- User responsible for data creation: Define the user responsible for creating data received from this CSV feed. Best practice is to dedicate one user per source for organizational clarity. Please [see the section "Best practices" below](import-automated.md#best-practices-section) for more information.
 - Import from date: Specify the date of the oldest data to retrieve. Leave the field empty to import everything.
 
 ![CSV feeds creation: prior CSV mapper test](assets/csv-feeds-creation.png)
```

Comment:
link broken

Reply:
@Jipegien, in production, the link is working fine : https://docs.opencti.io/latest/usage/import-automated/?h=import#csv-feeds